### PR TITLE
Update README_cn.md

### PR DESCRIPTION
**Pull request by OpenVINO-dev-contest**

### CODE REVIEW

**Comment by adrianboguszewski** [comment]
@OpenVINO-dev-contest, please wait with this PR for #944

**Comment by eaidova** [comment]
@OpenVINO-dev-contest  @zhuo-yoyowz is it ready to merge? I need fix for fixing typo in blip notebook name in readme.md (it is included in your PR) and want to understand if should I create a separate request for It or just wait until you finish review of these changes

**Comment by zhuo-yoyowz** [comment]
> @OpenVINO-dev-contest @zhuo-yoyowz is it ready to merge? I need fix for fixing typo in blip notebook name in readme.md (it is included in your PR) and want to understand if should I create a separate request for It or just wait until you finish review of these changes

@eaidova Do you mean you need to change the blip notebook name in the English readme.md? If so, we should make changes to the Chinese version readme after you finish changing in the EN version, and then merge the Chinese version after.

**Comment by eaidova** [comment]
@zhuo-yoyowz 
>>@eaidova Do you mean you need to change the blip notebook name in the English readme.md?
yes, I mean this, but if this PR is ready, then I prefer to just merge it instead postpone changes in Chinese readme and rebase PR

File: README_cn.md [hunk 1]
```diff
@@ -127,7 +127,10 @@ Jupyter notebooks 分为四个大类，选择一个跟你需求相关的开始
...
 |[231-instruct-pix2pix-image-editing](notebooks/231-instruct-pix2pix-image-editing/)<br>| 利用InstructPix2Pix进行图像编辑 | <img src=https://user-images.githubusercontent.com/29454499/219943222-d46a2e2d-d348-4259-8431-37cf14727eda.png width=225> |
-
+|[232-clip-language-saliency-map](notebooks/232-clip-language-saliency-map/)<br>|  基于CLIP和OpenVINO™的视觉语言显著性检测 | <img src=https://user-images.githubusercontent.com/29454499/218967961-9858efd5-fff2-4eb0-bde9-60852f4b31cb.JPG width=225> | 
+|[233-blip-visual-language-processing](notebook/233-blip-visual-language-processing/)<br>| 基于BLIP和OpenVINO™的视觉问答与图片注释 | <img src=https://user-images.githubusercontent.com/29454499/221933762-4ff32ecb-5e5d-4484-80e1-e9396cb3c511.png width=225> |
+|[234-encodec-audio-compression](notebooks/234-encodec-audio-compression/)<br>| # 基于EnCodec和OpenVINO™的音频压缩 | <img src=https://github.com/facebookresearch/encodec/raw/main/thumbnail.png width=225> |
+|[235-controlnet-stable-diffusion](notebooks/235-controlnet-stable-diffusion/)<br>| # 使用ControlNet状态调节和OpenVINO™ 实现文字生成图片 | <img src=https://user-images.githubusercontent.com/29454499/224541412-9d13443e-0e42-43f2-8210-aa31820c5b44.png width=225> |
```
Comment (zhuo-yoyowz) [inline README_cn.md:133]:
```suggestion
|[235-controlnet-stable-diffusion](notebooks/235-controlnet-stable-diffusion/)<br>| # 使用ControlNet状态调节Stable Diffusion 实现文字生成图片 | <img src=https://user-images.githubusercontent.com/29454499/224541412-9d13443e-0e42-43f2-8210-aa31820c5b44.png width=225> |
```

**Comment by OpenVINO-dev-contest** [comment]
I think it ready to be merged

**Review by zhuo-yoyowz — APPROVED** [review]

**Review by eaidova — APPROVED** [review]

**Pull request merged**